constants/constant.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"math"
```

Comment:
The idea behind this import is to demonstrate how fmt.Println() handles different types as a result of an operation